pkg/controller/cell.go

```diff
@@ -7,35 +7,65 @@ import (
 	"github.com/spf13/pflag"
 
 	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/metrics"
+	"github.com/cilium/cilium/pkg/metrics/metric"
 )
 
 const (
 	controllerGroupMetrics = "controller-group-metrics"
+
+	// labelControllerGroupName is the label used
+	// to identify controller-specific metrics
+	labelControllerGroupName = "group_name"
 )
 
-var ControllerGroupMetrics = map[string]bool{}
+var (
+	// GroupMetricEnabled is populated with the set of ControllerGroups for which metrics are enabled
+	GroupMetricEnabled = map[string]bool{}
```

Comment:
Should this be exported, seems dangerous if someone updates this map from another package async.

Reply:
agreed